pkg/util/env.go

```diff
@@ -16,6 +20,16 @@ const filebeatImage = "FILEBEAT_IMAGE"
 const journalbeatImage = "JOURNALBEAT_IMAGE"
 const weblogicOperatorImage = "WEBLOGIC_OPERATOR_IMAGE"
 const fluentdImage = "FLUENTD_IMAGE"
+const cohMicroRequestMemory = "COH_MICRO_REQUEST_MEMORY"
```

Comment:
Are these env vars that the user would specify in the model file?  If not then where?  If they are then we should use the standard kubernetes resource request schema rather than env vars.

Reply:
The env vars are specified in the helm chart, like this one here:  https://github.com/verrazzano/verrazzano/blob/guzhang/vz830_v/install/chart/values.yaml.
They are system-wide settings, not for any model or binding.